### ISSUE TITLE
Some images are not working. Even on your online sample page

### ISSUE THREAD

**Issue by yeo7k** [scottcheng/cropit#219]
![avatar2](https://cloud.githubusercontent.com/assets/4844269/20863802/259391aa-b9af-11e6-8cb2-900b7e5c6b7c.png)

**Comment by kshitijdeota** [comment]
+1

**Comment by LukeMcLachlan** [comment]
is that because the image dimensions of the image that you're trying to upload are too small? I have noticed on the samples page that if images are too small the error message, telling you that the dimensions are too small, doesn't appear because of the z-index of the error message.

**Comment by kshitijdeota** [comment]
Yes, if the image dimensions are too small and also sometimes when the exif data is absent/stripped out or not recognized properly
